### DISCUSSION TITLE
多人联机时自动下载对方的版本与mod

### DISCUSSION THREAD

**Discussion by YricOTF** [Meloong-Git/PCL#481] [提问] [answered]
等联机功能出来了建议加一个

**Comment by WForst-Breeze** [comment]
如有新功能建议，请提交[新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)
（但您所提及的内容我**个人认为**几乎……不太可能加……不论是从技术层面还是从必要程度上qwq）

**Comment by ForgottenZ** [comment]
这玩意确实对玩家来说友好了不少，但是下载速度方面没准还不如你直接拿个QQ跟对面传快吧。。

**Comment by LTCatt** [comment, marked as answer]
不考虑制作此功能，它太超出启动器的范畴了 :(

**Comment by Chiloven945** [comment]
这怕是模组加载器的活......

**Comment by ruattd** [comment]
如果你是服主你可能需要 serverSync
如果你是玩家还不如自己打包传给对方...坐等 PCL2 的整合包导出吧...
